.github/workflows/ci.yml

```diff
@@ -120,3 +120,60 @@ jobs:
         curl -L https://github.com/ndmitchell/hlint/releases/download/v${HLINT_VERSION}/hlint-${HLINT_VERSION}-x86_64-linux.tar.gz --output hlint.tar.gz
         tar -xvf hlint.tar.gz
         ./hlint-${HLINT_VERSION}/hlint src/ test/
+
+  stan:
+    name: stan / ghc ${{ env.GHC_VERSION }}
```

Comment:
In that case, let's leave just `stan`, without the GHC version. I don't like to have many places that require manual updates as it's really easy to forget to update them.